atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/SweepCommand.java

```diff
@@ -0,0 +1,170 @@
+    String row;
+
+    @Option(name = {"-a", "--all"},
+            description = "Sweep all tables")
+    boolean sweepAllTables;
+
+	@Override
+	protected int execute(final AtlasDbServices services) {
+        AtlasDbConfig config = services.getServerConfig();
+        SweepTaskRunner sweepRunner = services.getSweepTaskRunner();
+
+        if (!((namespace != null) ^ (table != null) ^ sweepAllTables)) {
+            System.err.println("Specify one of --namespace, --table, or --all options.");
+            return 1;
+        }
+        if ((namespace != null) && (row != null)) {
+            System.err.println("Cannot specify a start row (" + row + ") when sweeping multiple tables (in namespace " + namespace + ")");
+            return 1;
+        }
+
+        Map<String, Optional<byte[]>> tableToStartRow = Maps.newHashMap();
+
+        if ((table != null)) {
+            Optional<byte[]> startRow = Optional.of(new byte[0]);
+            if (row != null) {
+                startRow = Optional.of(decodeStartRow(row));
+            }
+            tableToStartRow.put(table, startRow);
+        } else if (namespace != null) {
+            Set<String> tablesInNamespace = Sets.filter(services.getKeyValueService().getAllTableNames(),
+                    Predicates.containsPattern("^" + namespace + "\\."));
+            for (String table : tablesInNamespace) {
+                tableToStartRow.put(table, Optional.of(new byte[0]));
+            }
+        } else if (sweepAllTables) {
+            tableToStartRow.putAll(
+                    Maps.asMap(
+                            Sets.difference(services.getKeyValueService().getAllTableNames(), AtlasDbConstants.hiddenTables),
+                            Functions.constant(Optional.of(new byte[0]))));
+        }
+
+        for (Map.Entry<String, Optional<byte[]>> entry : tableToStartRow.entrySet()) {
+            final String table = entry.getKey();
+            Optional<byte[]> startRow = entry.getValue();
+
+            final AtomicLong cellsExamined = new AtomicLong();
+            final AtomicLong cellsDeleted = new AtomicLong();
+
+            while (startRow.isPresent()) {
+                Stopwatch watch = Stopwatch.createStarted();
+                SweepResults results = sweepRunner.run(table, config.getSweepBatchSize(), startRow.get());
+                System.out.println(String.format("Swept from %s to %s in table %s in %d ms, examined %d unique cells, deleted %d cells.",
+                        encodeStartRow(startRow), encodeEndRow(results.getNextStartRow()),
+                        table, watch.elapsed(TimeUnit.MILLISECONDS),
+                        results.getCellsExamined(), results.getCellsDeleted()));
+                startRow = results.getNextStartRow();
+                cellsDeleted.addAndGet(results.getCellsDeleted());
+                cellsExamined.addAndGet(results.getCellsExamined());
+                try {
+                    Thread.sleep(config.getSweepPauseMillis());
```

Comment:
It makes perfect sense for the background sweeper (with the intent that we go slowly and don't greatly affect normal usage perf); there was a support ticket that forced me to add the same thing into the PG CLI for a customer that was experiencing Cassandra cluster instability while doing some questionable amount of work at once (I think it was simultaneous backups / new resync / sweeping / running out of DB disk space / normal stack usage)